src/applications/vaos/services/appointment/transformers.js

```diff
@@ -474,7 +474,7 @@ export function transformConfirmedAppointments(appointments) {
   return appointments.map(appt => transformConfirmedAppointment(appt));
 }
 
-function getTypeOfCareById(id) {
+export function getTypeOfCareById(inputId) {
```

Comment:
Is this function the same as 'getTypeOfCare(serviceType)' in ../vaos/utils/appointment.js?

Reply:
same as above